pom.xml

```diff
@@ -5402,7 +5402,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
-          <version>2.10.3</version>
+          <version>3.2.0</version>
```

Comment:
There's a second 2.10.3 mention in the same pom, see d60f1ec0ddb0fd2dfef1bca97ab93a130fe8f0ec

Reply:
This one https://github.com/nuxeo/nuxeo/pull/4328/files#diff-600376dffeb79835ede4a0b285078036L6461?

Reply:
Indeed, i missed the change :+1: